INSTALL.md

```diff
@@ -30,9 +30,10 @@ If you'd like to create a VM with BLT, you will require the following additional
 
         brew tap caskroom/cask
         brew install ansible
+        brew upgrade ansible
```

Comment:
It looks like in this case, you might have installed Ansible a while back, and just need to update it on your system (same as you'd need to do with VirtualBox or Vagrant).

Homebrew usually has the latest version within a day or less of official release... and checking now, it looks like this is the version you'll get if you install `ansible`:

```
$ brew info ansible
ansible: stable 2.2.0.0 (bottled), HEAD
```